### DISCUSSION TITLE
last name and first name with german umlaut (äöü) are not encoded correctly

### DISCUSSION THREAD

**Discussion by volkerrichert** [keycloak/keycloak#10898] [Q&A]
Hi! I need some help.

We set up keycloak 16.1.1 and use MS Office 365 AzureAD as an OpenID Connect federation provider. It works great!

But in Germany, we have some special chars like "äüöß" in some names like "Müller" oder "Schröder".

When a new user logs in the first time his names are set to some broken strings like "Schr��der" or "M��ller". It looks like an encoding issue, but i didn't find any way to configure it. 

Any hint?

THX,

Volker

**Comment by cgeorgilakis** [comment]
In our Keycloak this letters - including Greek names - are working well.
Do you have problem in database encoding?

**Comment by volkerrichert** [comment]
hm. I run auth using a helm chart from bitnami. How need the db encoding has to be configured?
